src/streamlink_cli/main.py

```diff
@@ -349,7 +349,7 @@ def handle_stream(plugin, streams, stream_name):
 
             console.msg("{0}", cmdline)
         else:
-            console.exit("The stream specified cannot be translated to a command")
+            console.exit("The specified stream cannot be translated to a command")
```

Comment:
`The specified stream cannot be translated into a command.`
(translated into + full stop)